tests/resource_user.tftest.hcl

```diff
@@ -0,0 +1,65 @@
+mock_provider "opslevel" {
+  alias  = "fake"
+  source = "./mock_resource"
+}
+
+run "resource_user" {
+  providers = {
+    opslevel = opslevel.fake
+  }
+
+  assert {
+    condition     = opslevel_user.mock_user.email == "mock_user@mock.com"
+    error_message = "wrong email for opslevel_user"
+  }
+
+  assert {
+    condition     = opslevel_user.mock_user.id != null && opslevel_user.mock_user.id != ""
+    error_message = "opslevel_user id should not be empty"
+  }
+
+  assert {
+    condition     = opslevel_user.mock_user.last_updated == "2022-02-24T13:50:07Z"
+    error_message = "wrong last_updated timestamp for opslevel_user"
```

Comment:
Seems unnecessary to test this

Reply:
It asserts the `last_updated` field exists, which is most of what we're testing at this point